addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -192,3 +193,75 @@ test("mention suggestions in thread match channel restrictions", async () => {
     await contains(".o-mail-Composer-suggestion", { text: "Mitchell Admin" });
     await contains(".o-mail-Composer-suggestion", { text: "p1" });
 });
+
+test("sub channel is muted (opacity-50) when mute is active", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    await start();
+    await openDiscuss(channelId);
+    await contains(".o-mail-Discuss-threadName", { value: "General" });
+    await click("button[title='Threads']");
+    await click("button[aria-label='Create Thread']");
+    await contains(".o-mail-Discuss-threadName", { value: "New Thread" });
```

Comment:
```js
await contains(".o-mail-Discuss-threadName", { value: "New Thread" });
```
is also unnecessary, as this is also asserted by another test.

We avoid to put too much assertion, because while this look like a good idea at first (we cover a lot of things), in practice tests are asserting far too much the same thing, and it becomes a real pain to adapt all the same assertions. Besides, the actual part that is uniquely important to the test is harder to spot.